src/User32.Desktop/User32+SystemMetric.cs

```diff
@@ -0,0 +1,628 @@
+﻿// Copyright (c) to owners found in https://github.com/AArnott/pinvoke/blob/master/COPYRIGHT.md. All rights reserved.
+// Licensed under the MIT license. See LICENSE.txt file in the project root for full license information.
+
+namespace PInvoke
+{
+    using System;
+    using System.Runtime.InteropServices;
+
+    /// <content>
+    /// Contains the <see cref="SystemMetric"/> nested type.
+    /// </content>
+    public partial class User32
+    {
+        /// <summary>
+        /// Flags used in the <see cref="GetSystemMetrics"/> function
+        /// </summary>
+        public enum SystemMetric
```

Comment:
We should probably call this `GetSystemMetricsFlags` then to fit the convention.